src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLQueryTypeImpl.java

```diff
@@ -532,6 +540,126 @@
     };
   }
 
+  @Override
+  public DataFetcher<Iterable<VehicleRentalStation>> vehicleRentalStations() {
+    return environment -> {
+      VehicleRentalStationService vehicleRentalStationService = getRoutingService(environment)
+              .getVehicleRentalStationService();
+
+      if (vehicleRentalStationService == null) {
+        return null;
+      }
+
+      var args = new LegacyGraphQLTypes.LegacyGraphQLQueryTypeVehicleRentalStationsArgs(
+              environment.getArguments());
+
+      if (args.getLegacyGraphQLIds() != null) {
+        ArrayListMultimap<String, VehicleRentalStation> vehicleRentalStations =
+                vehicleRentalStationService.getVehicleRentalStations()
```

Comment:
Should there be a helper method for getting on vehicles/stations in VehicleRentalStationService and/or should we store the vehicles and the stations in separate Maps there?

Reply:
I think a helper method would suffice

Reply:
Ok that's done now.